src/common/field/index.js

```diff
@@ -54,16 +54,16 @@ const FieldMixin = {
     /** @inheritdoc */
     render() {
         const {error} = this.state;
-        const {FieldComponent, InputLabelComponent, domain, codeResolver, searcher, isRequired, values, hasLabel, isEdit} = this.props;
+        const {FieldComponent, InputLabelComponent, domain, codeResolver, searcher, keyResolver, querySearcher, isRequired, values, hasLabel, isEdit} = this.props;
         const isCustomComponent = FieldComponent || InputLabelComponent;
-        const {autocomplete, label, input, select, display} = this;
+        const {autocomplete, autocompleteSelect, label, input, select, display} = this;
         return (
             <div className='mdl-grid' data-domain={domain} data-focus='field' data-mode={isEdit ? 'edit' : 'consult'} data-required={isRequired} data-valid={!error}>
                 {isCustomComponent && this._renderFieldComponent()}
                 {!isCustomComponent && hasLabel && label()}
                 {!isCustomComponent &&
                     <div className ={`${this._getContentGridClassName()}`} data-focus='field-value-container'>
-                        {codeResolver && searcher ? autocomplete() : isEdit ? (values ? select() : input()) : display()}
+                        {codeResolver && searcher ? autocomplete() : keyResolver && querySearcher ? autocompleteSelect() : isEdit ? (values ? select() : input()) : display()}
```

Comment:
That's intentionnal, I want to keep it dirty to press the urge for the refactor we are working on :)